terraform/configure.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+
+'''
+This script will configure the terraform environment based on the active
+workspace.
+
+Due to a terraform limitation, modules can not be scaled with the "count"
+keyword the same way resource can.  This leads to a great deal of copying and
+pasting in order to get provisioners and other modules to execute correctlyself.
+
+This script can be retired once this limitation is removed from Terraform.
+See: https://github.com/hashicorp/terraform/issues/953
+
+To create a unique configuration for your own workspace edit the
+WORKSPACE_CONFIGS constant below.
+'''
+
+import sys
```

Comment:
I like imports to be sorted in three groups in this order:
* Python built-ins
* External dependencies
* Local dependencies

Then, within each group, I like the imports to be sorted alphabetically.  This makes it easy to find imports as the list grows in length.

What do you think about doing that here?